### ISSUE TITLE
Enlarge script area

### ISSUE THREAD

**Issue by sfederici** [ScratchAddons/ScratchAddons#4536]
### Is your request related to a problem? Please describe.

The sprite names of the sprite thumbnails in the last line of the sprite area are not visible. Even using the scrollbar you cannot scroll down enough to see the sprite names.

### Describe the solution you'd like

Enlarge the sprite area, so to allow to scroll down enough to see all sprite names.
 
### Describe alternatives you've considered

Adding a sprite name tooltip for the thumbnails in the sprite area

**Comment by WorldLanguages** [comment]
@sfederici Could you post a screenshot/video?

**Comment by sfederici** [comment]
https://user-images.githubusercontent.com/2651868/167270258-a6947450-b8d8-4ef5-a039-c9dd771e794f.mov

As you can see, if the window size is 100% the names of the last sprites are not visible. You can make them visible if you shrink the windows size (but this is not what I want)

**Comment by WorldLanguages** [comment]
@sfederici Are you able to reproduce this on a browser? Or is it only Scratch/TurboWarp Desktop?

**Comment by sfederici** [comment]
<img width="1344" alt="Schermata 2022-05-07 alle 22 47 34" src="https://user-images.githubusercontent.com/2651868/167271213-9a85ba08-1a13-47c9-ba5a-8be79ebffa22.png">
 Here you go

**Comment by WorldLanguages** [comment]
@sfederici That's TurboWarp, can you reproduce on scratch.mit.edu with the Scratch Addons extension disabled?

**Comment by Norbiros** [comment]
Scroll button shows when you are hovering over it (in turbowarp).
In scratch we have ugly scroll  bar that is visible always (no matter where your mouse is)

BTW, we can make addon that makes scroll bar better.

**Comment by sfederici** [comment]
> @sfederici That's TurboWarp, can you reproduce on scratch.mit.edu with the Scratch Addons extension disabled?

No

**Comment by sfederici** [comment]
> Scroll button shows when you are hovering over it (in turbowarp). In scratch we have ugly scroll bar that is visible always (no matter where your mouse is)
> 
> BTW, we can make addon that makes scroll bar better.



> > @sfederici That's TurboWarp, can you reproduce on scratch.mit.edu with the Scratch Addons extension disabled?
> 
> No



> Scroll button shows when you are hovering over it (in turbowarp). In scratch we have ugly scroll bar that is visible always (no matter where your mouse is)

Yes, they do, but I cannot scroll enough to make the sprites names of the sprites in the last line visible

**Comment by Norbiros** [comment]
> > @sfederici That's TurboWarp, can you reproduce on scratch.mit.edu with the Scratch Addons extension disabled?
> 
> No. So, is this a bug due to some addon or to turbowarp?

Probably to addon that is in SA and Turbowarp... Can you try disabling all of your addons one by one and testing when this bug is gone? Now it is difficult to fix, because we don't know where bug is...

Also, I can't reproduce this bug. Maybe it depends on screen size?

**Comment by sfederici** [comment]
> Can you try disabling all of your addons one by one and testing when this bug is gone? Now it is difficult to fix, because we don't know where bug is...

I have many addons enabled. I will try it as soon as possible

> Also, I can't reproduce this bug. Maybe it depends on screen size?

It doesn't seem so. Here you have 4 different screen resolutions. They have all the same bug

<img width="1152" alt="Schermata 2022-05-08 alle 17 40 25" src="https://user-images.githubusercontent.com/2651868/167303871-4eb2e49a-d0ee-4484-bd72-ec2811e89f34.png">
<img width="1536" alt="Schermata 2022-05-08 alle 17 40 08" src="https://user-images.githubusercontent.com/2651868/167303875-5664d63b-66cc-4f04-b0b2-71638f8dd9aa.png">
<img width="1792" alt="Schermata 2022-05-08 alle 17 39 52" src="https://user-images.githubusercontent.com/2651868/167303876-efb9342b-2508-4c32-af32-da1c53f6ba09.png">
<img width="2048" alt="Schermata 2022-05-08 alle 17 39 31" src="https://user-images.githubusercontent.com/2651868/167303879-cb205498-eafc-4409-9f76-99ce8b560baf.png">

**Comment by GarboMuffin** [comment]
This appears to be a Scratch bug that happens when the language is set to any of these languages: https://github.com/LLK/scratch-gui/blob/7ecec48bf9df6671bc2f1e4550304debee388483/src/lib/locale-utils.js#L6-L18

**Comment by sfederici** [comment]
> This appears to be a Scratch bug that happens when the language is set to any of these languages: https://github.com/LLK/scratch-gui/blob/7ecec48bf9df6671bc2f1e4550304debee388483/src/lib/locale-utils.js#L6-L18

Thanks. Comparing the Italian and Englsh GUI I found out that the Sprite Info area is taller in Italian:
- the "Sprite" label in the first line of the Italian Sprite Info area is above the sprite name box instead of inline. This is weird, as all the labels in the first line are identical to the English ones
- the "Show" label is slightly shorter than the Italian "Mostra"
- the "Size" label is shorter than the Italian "Dimensione"
- the "Direction" label is almost identical to the Italian "Direzione"
As there is enough space around the labels, it seems to me that even the Italian ones should fit on a single line. Maybe the label areas in the Sprite Info area are costrained to fixed widths, so that if the space for one of the labels is slightly larger than the planned one all the labels go above the boxes instead of inline?
